webpack.config.js

```diff
@@ -47,15 +48,15 @@ module.exports = (env, argv) => ({
                                     "liquid",
                                     "markdown",
                                     "markup",
-                                    "php"
+                                    "php",
                                 ],
                                 plugins: ["line-numbers"],
                                 theme: "default", //"coy","tomorrow"
-                                css: true
-                            }
-                        ]
-                    ]
-                }
+                                css: true,
+                            },
+                        ],
```

Comment:
Some new comma madness? 😆

Reply:
Eurgh - that's my auto-formater. Prettier changed its default settings. Work prefers trailing commas (makes diffs cleaner going forward), so they're not in my Prettier overrides.

Thoughts? Keep trailing commas as standard?

Reply:
Sure, I'm happy to keep them